dataactvalidator/config/sqlrules/c20_award_financial.sql

```diff
@@ -1,137 +1,182 @@
-    op.obligations_incurred_by_pr_cpe,
-    op.ussgl487100_downward_adjus_cpe,
-    op.ussgl497100_downward_adjus_cpe,
-    op.ussgl487200_downward_adjus_cpe,
-    op.ussgl497200_downward_adjus_cpe,
-    op.deobligations_recov_by_pro_cpe
-HAVING SUM(af.ussgl480100_undelivered_or_fyb) < op.ussgl480100_undelivered_or_fyb
-    OR SUM(af.ussgl480100_undelivered_or_cpe) < op.ussgl480100_undelivered_or_cpe
-    OR SUM(af.ussgl483100_undelivered_or_cpe) < op.ussgl483100_undelivered_or_cpe
-    OR SUM(af.ussgl488100_upward_adjustm_cpe) < op.ussgl488100_upward_adjustm_cpe
-    OR SUM(af.obligations_undelivered_or_fyb) < op.obligations_undelivered_or_fyb
-    OR SUM(af.obligations_undelivered_or_cpe) < op.obligations_undelivered_or_cpe
-    OR SUM(af.ussgl490100_delivered_orde_fyb) < op.ussgl490100_delivered_orde_fyb
-    OR SUM(af.ussgl490100_delivered_orde_cpe) < op.ussgl490100_delivered_orde_cpe
-    OR SUM(af.ussgl493100_delivered_orde_cpe) < op.ussgl493100_delivered_orde_cpe
-    OR SUM(af.ussgl498100_upward_adjustm_cpe) < op.ussgl498100_upward_adjustm_cpe
-    OR SUM(af.obligations_delivered_orde_fyb) < op.obligations_delivered_orde_fyb
-    OR SUM(af.obligations_delivered_orde_cpe) < op.obligations_delivered_orde_cpe
-    OR SUM(af.ussgl480200_undelivered_or_fyb) < op.ussgl480200_undelivered_or_fyb
-    OR SUM(af.ussgl480200_undelivered_or_cpe) < op.ussgl480200_undelivered_or_cpe
-    OR SUM(af.ussgl483200_undelivered_or_cpe) < op.ussgl483200_undelivered_or_cpe
-    OR SUM(af.ussgl488200_upward_adjustm_cpe) < op.ussgl488200_upward_adjustm_cpe
-    OR SUM(af.gross_outlays_undelivered_fyb) < op.gross_outlays_undelivered_fyb
-    OR SUM(af.gross_outlays_undelivered_cpe) < op.gross_outlays_undelivered_cpe
-    OR SUM(af.ussgl490200_delivered_orde_cpe) < op.ussgl490200_delivered_orde_cpe
-    OR SUM(af.ussgl490800_authority_outl_fyb) < op.ussgl490800_authority_outl_fyb
-    OR SUM(af.ussgl490800_authority_outl_cpe) < op.ussgl490800_authority_outl_cpe
-    OR SUM(af.ussgl498200_upward_adjustm_cpe) < op.ussgl498200_upward_adjustm_cpe
-    OR SUM(af.gross_outlays_delivered_or_fyb) < op.gross_outlays_delivered_or_fyb
-    OR SUM(af.gross_outlays_delivered_or_cpe) < op.gross_outlays_delivered_or_cpe
-    OR SUM(af.gross_outlay_amount_by_awa_fyb) < op.gross_outlay_amount_by_pro_fyb
-    OR SUM(af.gross_outlay_amount_by_awa_cpe) < op.gross_outlay_amount_by_pro_cpe
-    OR SUM(af.obligations_incurred_byawa_cpe) < op.obligations_incurred_by_pr_cpe
-    OR SUM(af.ussgl487100_downward_adjus_cpe) < op.ussgl487100_downward_adjus_cpe
-    OR SUM(af.ussgl497100_downward_adjus_cpe) < op.ussgl497100_downward_adjus_cpe
-    OR SUM(af.ussgl487200_downward_adjus_cpe) < op.ussgl487200_downward_adjus_cpe
-    OR SUM(af.ussgl497200_downward_adjus_cpe) < op.ussgl497200_downward_adjus_cpe
-    OR SUM(af.deobligations_recov_by_awa_cpe) < op.deobligations_recov_by_pro_cpe;
\ No newline at end of file
+SELECT ussgl480100_undelivered_or_fyb_sum_c,
```

Comment:
What do you think about returning the related b and c file columns together so that they will display next to one another on the warning report? It might make it easier for people to see--of this long field list--which are the ones causing the problems. For example:
```sql
SELECT 
    ussgl480100_undelivered_or_fyb_sum_c,
    ussgl480100_undelivered_or_fyb_sum_b,
    etc
```